_includes/package-grid.html

```diff
@@ -45,10 +45,10 @@ <h3 class="card__title no_toc" itemprop="headline">
             {% if apackage.joss %}
                 <li><a href="{{ apackage.archive }}" rel="permalink"><i class="fas fa-bookmark fa-fw"></i> JOSS Approved</a></li>
             {% endif %}
-            {% if apackage.astropy %}
+            {% if apackage.partners contains "astropy" %}
```

Comment:
@erik-whiting i'm sorry what i asked you to do is exactly what you did. let's go ahead and merge this knowing the other changes will get overwritten in the next contributor update.